src/hash_impl.h

```diff
@@ -163,6 +163,20 @@ static void secp256k1_sha256_finalize(secp256k1_sha256 *hash, unsigned char *out
     memcpy(out32, (const unsigned char*)out, 32);
 }
 
+/* Initializes a sha256 struct and writes the 64 byte string
+ * SHA256(tag)||SHA256(tag) into it. The taglen should be less than or equal to
```

Comment:
Why should the tag be at most 64 bytes long?

Reply:
because brainfart, fixed